README.md

```diff
@@ -1 +1,54 @@
+<!-- markdownlint-configure-file
+{
+  "MD013": {
+    "line_length": 120,
+    "strict": true
+  }
+}
+-->
```

Comment:
I'll either remove this or add markdownlint to CI, this was mainly to help me be consistent

If you're wondering why 120: 100 was too short, 120 still fits in a side-by-side diff on github on my screen :smile:

Reply:
removed